src/pages/Sign/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { TopBar } from 'components/TopBar'
+import * as React from 'react'
+import styled from 'react-emotion'
+import { SignUp } from './sections/SignUp'
+
+const Wrapper = styled('div')({})
+
+export const Sign: React.SFC<{}> = () => (
```

Comment:
just fyi, the generic defaults to `{}` so  `React.SFC` is sufficient. Not that it matters much but you can save a few keystrokes here and there :D